issue-156: change plugin id to Grafana naming convention (#161)
VictoriaMetrics/victorialogs-datasource: CHANGELOG.md

## tip

> **Update note 1: in the new version of the plugin was updated plugin id name. The new plugin id name is `victoriametrics-logs-datasource`.
> Those changes require updating the field `allow_loading_unsigned_plugins` in the `grafana.ini`or `defaults.ini` configuration file. 
> Example `allow_loading_unsigned_plugins = victoriametrics-logs-datasource`. In case of usage provision it is required to change `type` 
> field to `victoriametrics-logs-datasource`. After updating this field it needed to restart the Grafana server.**

Reviewer: What do you think about explicitly highlighting the Breaking Change? Here's the revised text:

⚠️ **Breaking Change: Plugin ID Updated**  

**Update Note 1:**  
In the new version of the plugin, the plugin ID has been updated. The new plugin ID is `victoriametrics-logs-datasource`. **This is a breaking change**: Grafana will treat this as a new plugin.  

- You must update the `allow_loading_unsigned_plugins` field in the `grafana.ini` or `defaults.ini` configuration file. **Example:**  
    ```ini  
    allow_loading_unsigned_plugins = victoriametrics-logs-datasource  
    ```

- If you are using provisioning, update the `type` field to `victoriametrics-logs-datasource` in your provisioning configuration.

- After making these changes, you must restart the Grafana server for the updates to take effect.  

```suggestion
⚠️ **Breaking Change: Plugin ID Updated**  

**Update Note 1:**  
In the new version of the plugin, the plugin ID has been updated. The new plugin ID is `victoriametrics-logs-datasource`. **This is a breaking change**: Grafana will treat this as a new plugin.  

- You must update the `allow_loading_unsigned_plugins` field in the `grafana.ini` or `defaults.ini` configuration file.  
    **Example:**  
    ```ini  
    allow_loading_unsigned_plugins = victoriametrics-logs-datasource  
    ```

- If you are using provisioning, update the `type` field to `victoriametrics-logs-datasource` in your provisioning configuration.

- After making these changes, you must restart the Grafana server for the updates to take effect.  
```
